Libplanet/Blocks/Block.cs

```diff
@@ -190,51 +191,19 @@ public byte[] ToBencodex(bool hash, bool transactionData)
         /// Executes every <see cref="IAction"/> in the
         /// <see cref="Transactions"/> and gets result states of each step of
         /// every <see cref="Transaction{T}"/>.
-        /// </summary>
-        /// <param name="accountStateGetter">An <see cref="AccountStateGetter"/>
-        /// delegate to get a previous state.
-        /// A <c>null</c> value, which is default, means a constant function
-        /// that returns <c>null</c>.</param>
-        /// <returns>Result states of immediately after
-        /// <see cref="IAction"/>s in each <see cref="Transaction{T}"/>
-        /// being executed.</returns>
-        [Pure]
-        public IEnumerable<IAccountStateDelta> EvaluateActions(
-            AccountStateGetter accountStateGetter = null
-        )
-        {
-            var pairs = EvaluateActionsPerTx(accountStateGetter);
-            int i = 0;
-            foreach (var (tx, evaluation) in pairs)
-            {
-                if (++i < tx.Actions.Count)
-                {
-                    continue;
-                }
-
-                yield return evaluation.OutputStates;
-                i = 0;
-            }
-        }
-
-        /// <summary>
-        /// Validates the integrity of the <see cref="Block{T}"/>.
         /// <para>It throws an <see cref="InvalidBlockException"/> or
         /// an <see cref="InvalidTxException"/> if there is any
         /// integrity error.</para>
-        /// <para>Otherwise it returns an <see cref="IAccountStateDelta"/>
-        /// which represents the final states and maintains the changes
-        /// from the states of the previous <see cref="Block{T}"/>.</para>
+        /// <para>Otherwise it returns an <see cref="ActionEvaluation{T}"/>
+        /// for each steps.</para>
         /// </summary>
         /// <param name="currentTime">The current time to validate
         /// time-wise conditions.</param>
         /// <param name="accountStateGetter">The getter of previous states.
         /// This affects the execution of <see cref="Transaction{T}.Actions"/>.
         /// </param>
-        /// <returns>An <see cref="IAccountStateDelta"/> of the states
-        /// right after all <see cref="Transaction{T}.Actions"/> of
-        /// <see cref="Transactions"/>, which maintains the changes from
-        /// the states of the previous <see cref="Block{T}"/>.</returns>
+        /// <returns>An <see cref="ActionEvaluation{T}"/> for each steps.
```

Comment:
```suggestion
        /// <returns>An <see cref="ActionEvaluation{T}"/> for each step.
```

Reply:
This sounds ambiguous too.

Reply:
It would be fine if we just copy a `<returns>` docs field of `Transaction<T>.EvaluateActionsGradually()` method and adjust some words.